code/modules/vehicles/interior/interactable/unique_interior_machinery.dm

```diff
@@ -0,0 +1,106 @@
+//This file contains unique miscellaneous modules and machinery for each type of APC
+
+//---------------------------MED APC------------------------
+
+//MED APC version of medevac stretcher for interior
+//can only be used when vehicle is not moving and only under allowed ceiling
+/obj/structure/bed/medevac_stretcher/vehicle
+	name = "medevac stretcher"
+	desc = "A medevac stretcher for rapid evacuation of an injured patient via dropship lift. Can only be used when vehicle is stationary and under glass or no ceiling. Accepts patients and body bags. Completely useless for surgery."
+	icon = 'icons/obj/vehicles/interiors/apc.dmi'
+	icon_state = "stretcher_down"
+	foldabletype = null
+	base_bed_icon = "stretcher"
+
+	unslashable = FALSE
+	indestructible = FALSE
+	unacidable = FALSE
```

Comment:
why do these need to be redefined?

Reply:
Because I was supposed to change them to TRUE after copy pasting, which I obviously didn't as usual.